CourseDevelopment/Week5/Chapter5.MD

```diff
@@ -0,0 +1,99 @@
+# Course Development Week 5
+Course Development Week 5 for the Open Science Course for physics students at graduate level. This document is based on the Syllabus.MD.
+
+Date: Dec/07/2022
+
+Name: Hendrik Snijder @hendelhendel
+
+Open tasks @SanliFaez @erikvansebille
++ Feedback on lecture-tutorial distribution of topics
++ Feedback on lecture format
++ Feedback on tutorial format
+
+## Goal: public engagement: story-telling and presentation; activism and trust in scientists
+After this week, one understands the role, responsibilities and interaction of scientists with scientific communities and the general public. 
+Students have to give a presentation about their other assignments using skills learned in the course. 
+Further, the studends are able to formulate their opinion in a discussion on the role a scientist must take in a public debate.
```

Comment:
'discussion' sounds rather vague? Make that more specific?

Reply:
What about:
Further, the studends are able to formulate their opinion on the role a scientist must take in society. Suported by arguments learned in class they fromulate statements. By taking into account different stakes in the public debate, they can refelect critically on their own on other viewpoint.

Reply:
Yep, good idea. Please incorporate that then
```suggestion
Further, the students are able to formulate their opinion on the role a scientist must take in society, supported by arguments learned in class. By taking into account different stakes in the public debate, they can individually reflect critically on other viewpoints.
```

Reply:
Not on their own viewpoint?